packages/sdks/src/components/render-block/render-block.lite.tsx

```diff
@@ -221,12 +225,20 @@ export default function RenderBlock(props: RenderBlockProps) {
            */}
           <For each={state.childrenWithoutParentComponent}>
             {(child) => (
-              <RenderBlock key={'render-block-' + child.id} block={child} />
+              <RenderBlock
+                key={'render-block-' + child.id}
+                block={child}
+                context={props.context}
```

Comment:
how important is this change (all the changes in here changing builder context to be a prob and not normal context)? I realized I was wrong and RenderBlock does get used by customers, and really should use normal context if possible

the main impact on users is they can define child blocks in their code, aka

```tsx
function myCustomComponent(props) {
  return <div>{props.someArray.map(block => <RenderBlock block={block} />)</div>
}
```

so we need to try to keep this the same

Reply:
1.) I needed to convert this to lite-component so that we don't have extra wrapper divs.
2.) Now that we have hostless we could convert this back, but we would have a lot of extra comments, which is not ideal.
3.) I think we should rename `RenderBlock` to `RenderBlockInternal`, which just calles `RenderBlock` but which can be used by our customers. This way we can have same API and not have extra junk in HTML.

Reply:
OK, looked into it deeper and I don't think this is an issue. 
- `<RenderBlocks>` Is exported and public. Its props interface did not change.
- `<RenderBlock>` is never exported (internal use only) so we can change its props without implication. So I am going to leave this as is.